downstream/assemblies/aap-on-azure/assembly-aap-azure-network-peering.adoc

```diff
@@ -0,0 +1,41 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-aap-azure-network-peering"]
+= Private network peering
+
+:context: aap-azure-install
+
+// [role="_abstract"]
+// You can use these instructions to install 
+
+Ansible Automation Platform on Azure is deployed into an independent managed resource group with its own Azure virtual network (VNET). 
```

Comment:
I could be wrong, but I noticed that on Microsoft's page, the capitalization is `Azure Virtual Network (VNet)` instead of `Azure virtual network (VNET)`.

Reply:
I used `sed` to do this:
```
 egrep -rl 'VNET' ./ | xargs -I@ sed -i '' 's/VNET/VNet/g' @
```
Weird notation in `xargs` because I ran this on a mac.